### PR TITLE
Rename debounce to throttle, alias debounce

### PR DESCRIPTION
**Pull request by ivancuric**
A short term solution. Long term I'd prefer to see a real debounce included, since I use that way more often personally.

### CODE REVIEW

File: src/decko.js [hunk 1]
```diff
@@ -46,11 +46,12 @@ let fns = {
 
 
 let memoize = multiMethod(fns.memoize),
-	debounce = multiMethod(fns.debounce),
+	debounce = multiMethod(fns.throttle),
+	throttle = multiMethod(fns.throttle),
```
Comment (developit) [inline src/decko.js]: would prefer `throttle = debounce,` here

Comment (ivancuric) [inline src/decko.js]: Like this?
```
let memoize = multiMethod(fns.memoize),
	throttle = multiMethod(fns.throttle),
	debounce = throttle,
	bind = multiMethod((f,c)=>f.bind(c), ()=>fns.bind);
```

**Comment by dolsem** [comment]
is there any update on this?

**Comment by dolsem** [comment]
I would like to submit a PR request for an actual debounce (backwards-compatible), but it makes sense to merge this first?

**Comment by remnev** [comment]
@developit do you still maintain this project? Could you somehow react here?

**Pull request closed without merging**